### ISSUE TITLE
Report warning when calling a method with an unannotated nullable references

### ISSUE THREAD

**Issue by raffaeler** [dotnet/roslyn#35981]
**Version Used**:  3.0.0-preview5-27626-15
Commit: 61f30f5a23
Visual Studio version: 16.2.0 Preview 1.0

**Steps to Reproduce**:

1.  A pre-C#8 class library there is a simple method (not annotated for nullable references)
```
public static int GetLength(string text) => text.Length; 
```

2.  Another project using `#nullable enable` gets green squiggles for nullable references and defines the following code in a method:
```
string? test = null;
Extensions.GetLength(test);
```

**Expected Behavior**:
Since `GetLength` has not being annotated with `NullableAttribute`, no squiggle is shown.
I would love to see the green squiggle on the `test` variable to be defensive on the new code. After all the exceptions bounces back to the caller which gets blamed not to have dereference protection.

**Actual Behavior**:
No warning on `test` is provided.

**Comment by gafter** [comment]
This is by design.  Since we don't know if `GetLength` accepts nulls or not, we do not warn.  Annotate the method if you want warnings on its use.

**Comment by raffaeler** [comment]
I wished at least in an optional warning, but thanks anyway for your answer.
